cluster-autoscaler/cloudprovider/clusterapi/clusterapi_unstructured.go

```diff
@@ -342,6 +345,27 @@ func (r unstructuredScalableResource) readInfrastructureReferenceResource() (*un
 	kind = fmt.Sprintf("%ss", strings.ToLower(kind))
 	gvk := schema.FromAPIVersionAndKind(apiversion, kind)
 	res := schema.GroupVersionResource{Group: gvk.Group, Version: gvk.Version, Resource: gvk.Kind}
+	for _, v := range []string{"list", "watch"} {
```

Comment:
ack, that all makes sense to me. i'd like to ask around internally a bit to see if anyone has ideas about the caching behavior.

Reply:
Finally made time to actually test this, it also works:

```
I0206 22:35:48.241966       1 clusterapi_unstructured.go:365] Unable to validate infrastructure reference authorization, error: <nil>
I0206 22:35:48.245691       1 clusterapi_provider.go:68] discovered node group: MachineDeployment/capz-e2e-uki1ko/capz-e2e-uki1ko-cas-capi-md-0 (min: 0, max: 5, replicas: 0)
I0206 22:35:48.245743       1 orchestrator.go:249] No similar node groups found
I0206 22:35:48.245768       1 orchestrator.go:291] Final scale-up plan: [{MachineDeployment/capz-e2e-uki1ko/capz-e2e-uki1ko-cas-capi-md-0 0->5 (max: 5)}]
I0206 22:35:48.245782       1 executor.go:147] Scale-up: setting group MachineDeployment/capz-e2e-uki1ko/capz-e2e-uki1ko-cas-capi-md-0 size to 5
I0206 22:35:48.245949       1 event_sink_logging_wrapper.go:48] Event(v1.ObjectReference{Kind:"ConfigMap", Namespace:"kube-system", Name:"cluster-autoscaler-status", UID:"108722ba-5563-474d-8a84-bcce98c3b05b", APIVersion:"v1", ResourceVersion:"75072", FieldPath:""}): type: 'Normal' reason: 'ScaledUpGroup' Scale-up: setting group MachineDeployment/capz-e2e-uki1ko/capz-e2e-uki1ko-cas-capi-md-0 size to 5 instead of 0 (max: 5)
```